CDCgov/prime-reportstream
# 6579 Allow for FHIR to HL7 schema extensions

This PR allows for FHIR to HL7 schema extensions.

Test Steps:
1. Build this branch.
2. Run the following command to generate an HL7 using the new CO extended schema: `./prime fhirdata --input-file src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir -s metadata/hl7_mapping/STLTs/CO/CO.yml`
3. Compare the output to `prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.hl7` and verify the overridden elements and HL7 fields are as defined in the CO.yml file

## Changes
- Created initial CO schema with data from the HCA's CO requirements plus RS CO settings
- Schemas can be extended to overwrite elements by name or add more elements.

## Checklist

### Testing

### Process

## Linked Issues
- #6579

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
